components/badge/overview.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Overview
+page_title: Badge Overview
+description: Discover the Badge component for Blazor. Learn how to add the component to your app and explore its configuration options, such as positioning and alignment.
+slug: badge-overview
+tags: telerik,blazor,badge,navbar
+published: True
+position: 0
+---
+
+# Blazor Badge Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/badge" target="_blank">Blazor Badge component</a> is a visual element that displays additional information, such as a status indicator, notification icon, short text, and others.
+
+The Badge comes with built-in customization features that allow the developer to fine-tune the [positioning and alignment]({%slug badge-position-alignment%}) of the component, change the [background color]({%slug badge-appearance%}#themecolor) and the [fill mode]({%slug badge-appearance%}#fillmode), and to enable or disable the cutout border.
+
+## Creating Blazor Badge
+
+1. Add the `<TelerikBadge>` tag to a container that the Badge will attach to.
+1. Use the `ChildContent` to add content.
+1. (optional) Customize the [appearance]({%slug badge-appearance%}) and fine-tune the [positioning and alignment]({%slug badge-position-alignment%}) of the Telerik Blazor Badge.
+
+>caption Basic configuration of the Blazor Badge
+
+````CSHTML
+<TelerikButton>
+    <TelerikBadge>12</TelerikBadge>
+   Notifications
+</TelerikButton>
+````
+
+## Positioning and Alignment
+
+You can control how the Badge is positioned relative to its associated container. [Read more about the Blazor Badge positioning...]({%slug badge-position-alignment%})
```

Comment:
There is no need to optimize our documentation pages for words like "read more about", so they should be outside the links. Overall, it's best if you can integrate links in regular text, rather than "read more" phrases.

```suggestion
You can control how the Badge is positioned relative to its associated container. Read more about the [Blazor Badge positioning...]({%slug badge-position-alignment%})
```